specification/cognitiveservices/data-plane/CustomVision/Prediction/stable/v3.0/Prediction.json

```diff
@@ -0,0 +1,917 @@
+            "schema": {
+              "$ref": "#/definitions/CustomVisionError"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Successful DetectImageUrlWithNoStore request": {
+            "$ref": "./examples/DetectImageUrlWithNoStore.json"
+          }
+        }
+      }
+    },
+    "/{projectId}/detect/models/{publishedModelName}/image/nostore": {
+      "post": {
+        "tags": [
+          "ImagePredictionApi"
+        ],
+        "summary": "Detect objects in an image without saving the result.",
+        "operationId": "DetectImageWithNoStore",
+        "consumes": [
+          "multipart/form-data",
+          "application/octet-stream"
+        ],
+        "produces": [
+          "application/json",
+          "text/json",
+          "application/xml",
+          "text/xml"
+        ],
+        "parameters": [
+          {
+            "name": "projectId",
+            "in": "path",
+            "description": "The project id.",
+            "required": true,
+            "type": "string",
+            "format": "uuid",
+            "x-nullable": false
+          },
+          {
+            "name": "publishedModelName",
+            "in": "path",
+            "description": "Specifies the name of the model to evaluate against.",
+            "required": true,
+            "type": "string",
+            "x-nullable": true
+          },
+          {
+            "name": "application",
+            "in": "query",
+            "description": "Optional. Specifies the name of application using the endpoint.",
+            "required": false,
+            "type": "string",
+            "x-nullable": true
+          },
+          {
+            "name": "imageData",
+            "in": "formData",
+            "description": "Binary image data.",
+            "required": true,
+            "type": "file"
+          },
+          {
+            "$ref": "#/parameters/ApiKey"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/ImagePrediction"
+            }
+          },
+          "default": {
+            "description": "Error response",
+            "schema": {
+              "$ref": "#/definitions/CustomVisionError"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Successful DetectImageWithNoStore request": {
+            "$ref": "./examples/DetectImageWithNoStore.json"
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "ImageUrl": {
+      "description": "Image url.",
+      "type": "object",
+      "properties": {
+        "url": {
+          "description": "Url of the image.",
+          "type": "string",
+          "x-nullable": true
+        }
+      },
+      "x-nullable": true
+    },
+    "ImagePrediction": {
+      "description": "Result of an image prediction request.",
+      "type": "object",
+      "properties": {
+        "id": {
```

Comment:
Is there an API that take the id as input? What can it be used for?

Reply:
Yes, the images stored during prediction calls with storage (classify/detectimage) are given ids that are used to reference them. They are used in CreateImagesFromPredictions/DeletePredictions.